core/src/main/java/org/elasticsearch/SecurityException.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.rest.RestStatus;
+
+import java.io.IOException;
+
+/**
+ * Generic security exception
+ */
+public class SecurityException extends ElasticsearchException {
```

Comment:
As far as common practice, i dont have an opinion. If it clashes with one from the jvm or really anything else in our classpath, then I think we should disambiguate. 

Otherwise IDE imports will do the wrong thing, I am sure of it: and I am not sure people review import statements well. I recently removed many usages of stuff like `com.carrotsearch.ant.tasks.junit4.dependencies.com.google.common.collect` from our codebase.